config/initializers/mime_types.rb

```diff
@@ -4,10 +4,12 @@
 # Mime::Type.register "text/richtext", :rtf
 
 # for azw3 files
-Mime::Type.register 'application/x-mobi8-ebook', :azw3
+#Mime::Type.extend 'application/x-mobi8-ebook', :azw3
```

Comment:
If needed, we can get the old version of this file from git, so no need to keep the old lines commented out